terraform/environments/extended-elms/output.tf

```diff
@@ -0,0 +1,3 @@
+output "vm_userpassword" {
+  value = module.iotedgevm.vm_userpassword
```

Comment:
What do we do with this? It's probably not the best idea to output it from the security perspective. So options that I see:
1. add Key Vault for storing it. But we don't have key vault in PowerShell script. And we will store there only this.
2. Do not output the VM password, but force user to provide it as a variable.

Reply:
Just a note, that the PS deploy script also outputs it.

Reply:
We agreed to not output the VM password.